packages/python/patch-no-opendir

```diff
@@ -0,0 +1,36 @@
+--- Modules/posixmodule.c
++++ Modules/posixmodule.c
+@@ -3663,7 +3663,9 @@
+         }
+ 
+         Py_BEGIN_ALLOW_THREADS
++#ifdef HAVE_OPENDIR
+         dirp = opendir(name);
```

Comment:
Is this safe to do? As in, is `dirp` initialised with `NULL` elsewhere? Maybe the entire function should be disabled? Same for the other changes in this file.